### PR TITLE
packaging/ubuntu-16.04/control: depend on fuse3 | fuse

### PR DESCRIPTION
**Pull request by simondeziel**
Fixes LP: #1792656

### CODE REVIEW

File: packaging/ubuntu-16.04/control [hunk 1]
```diff
@@ -14,6 +14,7 @@ Build-Depends: autoconf,
                dh-golang (>=1.7),
                debhelper (>= 9.20160709~) | dh-systemd,
                fakeroot,
+               fuse3 (>= 3.10.5-1) | fuse,
```
Comment (bboozzoo) [inline packaging/ubuntu-16.04/control:17]: Does it mean we can drop the dependency on libfuse now?

Comment (simondeziel) [reply packaging/ubuntu-16.04/control]: Oops, the dependency is a run time one, not a build time. I've now moved it where it belongs. Bottom line, libfuse is still needed when building.

**Review by bboozzoo — APPROVED** [review]
Thanks

**Review by mardy — APPROVED** [review]
LGTM, thanks!

**Review by mvo5 — APPROVED** [review]
Thank you

**Pull request merged**